css-two-column-layout/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row header">
+      <div class="row">
+        <h1>SCALE, SPEED, AGILITY</h1>
+      </div>
+      <div class="row">
+        <p>
+          We've dedicated thousands of man hours to making healthcare institutions in the Helium network run 10x more efficiently and deliver an unparalleled quality of care.
+        </p>
+      </div>
```

Comment:
There shouldn't ever be a `row` inside of a `row`. This means you probably need a `column-full` div to hold these rows